sdk/dotnet/Pulumi/Core/Output.cs

```diff
@@ -26,6 +26,12 @@ public static Output<T> CreateSecret<T>(T value)
         public static Output<T> CreateSecret<T>(Task<T> value)
             => Output<T>.CreateSecret(value);
 
+        public static Output<T> CreateUnknown<T>(T value)
```

Comment:
Docs are missing... We should definitely explain why this is needed. I'm inclined to put it into `OutputUtilities` too, given how niche the scenario is...

Reply:
Moved and documented.